src/runtime_src/core/edge/user/aie/aie.cpp

```diff
@@ -417,4 +417,117 @@ start_profiling_event_count(const std::string& port_name)
   return handle;
```

Comment:
Copyright needs to be updated (line 3).  Being that we are part of AMD now, the following addition should be made.
`Copyright (C) Advanced Micro Devices, Inc.`